doc/metadata/README.md

```diff
@@ -0,0 +1,20 @@
+Verify in-toto's supply chain with in-toto
+===========================================
+
+**Overview**  
+This document describes how to use in-toto to verify the software supply chains for the PyPi source of in-toto v1.0.1.
+
+The basic supply chains are laid out in "in_toto_sdist.layout" and consist of two steps:
+
+1. "git-tag" -- Sign the tag using the command "git tag --sign v1.0.1 -m "v1.0.1"".
+2. "create-sdist" -- Creates source distribution (.tar.gz) using products from (1) as materials.
+
+**Notes/Discussion**  
+
+- For the bootstrapping/chicken-egg situation, we assume that the user already has an older version of the in-toto locally.
+- For the first step, we want to explicitly allow all the files to make sure we tag all the materials files to sign.
+- Since we only sign the tag in the first step, no files will be changed and there will be no products. So we only need to add the "DISALLOW *" rule for the expected products.
```

Comment:
```suggestion
- Since we only tag a release in the first step, no files are changed and there will be no products. So we only need to add the "DISALLOW *" rule for the expected products.
```